cms/djangoapps/contentstore/config/waffle.py

```diff
@@ -22,10 +22,6 @@
     f'{WAFFLE_NAMESPACE}.enable_checklists_quality', __name__, LOG_PREFIX
 )
 
-SHOW_REVIEW_RULES_FLAG = CourseWaffleFlag(  # lint-amnesty, pylint: disable=toggle-missing-annotation
-    f'{WAFFLE_NAMESPACE}.show_review_rules', __name__, LOG_PREFIX
-)
-
```

Comment:
With SHOW_REVIEW_RULES_FLAG  removed, can you ensure that there's a unit test somewhere which validates that the new `show_review_rules` option works as intended?